audit-resolve.json

```diff
@@ -0,0 +1,94 @@
+{
+  "decisions": {
+    "755|hapi-swagger>handlebars": {
+      "madeAt": 0,
+      "decision": "ignore"
+    },
+    "788|hapi-swagger>json-schema-ref-parser>js-yaml": {
+      "madeAt": 0,
+      "decision": "ignore"
+    },
+    "788|hapi-swagger>swagger-parser>json-schema-ref-parser>js-yaml": {
+      "madeAt": 0,
+      "decision": "ignore"
+    },
+    "813|hapi-swagger>json-schema-ref-parser>js-yaml": {
+      "madeAt": 0,
+      "decision": "ignore"
+    },
+    "813|hapi-swagger>swagger-parser>json-schema-ref-parser>js-yaml": {
+      "madeAt": 0,
+      "decision": "ignore"
+    },
+    "788|istanbul>istanbul-api>js-yaml": {
+      "madeAt": 0,
+      "decision": "ignore"
+    },
+    "788|istanbul>js-yaml": {
+      "madeAt": 0,
+      "decision": "ignore"
+    },
+    "788|rewire>eslint>js-yaml": {
+      "madeAt": 0,
+      "decision": "ignore"
+    },
+    "788|standard>eslint>js-yaml": {
+      "madeAt": 0,
+      "decision": "ignore"
+    },
+    "788|tap-xunit>tap-parser>js-yaml": {
+      "madeAt": 0,
+      "decision": "ignore"
+    },
+    "813|istanbul>istanbul-api>js-yaml": {
```

Comment:
we dont use istanbul anymore

Reply:
It is used by nyc :D

Reply:
yeah but there is one for the istanbul on nyc, this is for the direct dependency